langwatch/src/components/evaluations/wizard/hooks/evaluation-wizard-store/slices/factories/llm-signature-node.factory.ts

```diff
@@ -10,7 +10,7 @@ const DEFAULT_LLM_CONFIG = {
 const DEFAULT_SIGNATURE_NODE_PROPERTIES: LlmSignatureNode = {
   type: "signature",
   id: "signature_node",
-  deletable: true,
+  deletable: false,
```

Comment:
only problem with doing like that, is that later on the person might want to go to the workflow directly to edit, and won't be able to. HOWEVER, I think it might actually be a good thing for now, we probably want the person to consciously "eject" from the wizard, and not being able to go back, otherwise if they go touching it left and right things will get weird

so maybe once they "eject" we remove this `deletable` 👍